server/run.sh

```diff
@@ -1,25 +1,28 @@
+#!/bin/bash
+
 for arg in "$@"
 do 
   case $arg in
     "--ci" )
-      python3 -m pip install -r requirements.txt;;
-    * )
+      ;;
```

Comment:
is this unused right now? Why not just delete it?

Reply:
Good call out. Wasn't sure if it was going to be needed yet, so left it in. Just forgot to take it out. Working on that now.

Reply:
Done :)